servers/src/main/java/tachyon/worker/block/BlockDataManager.java

```diff
@@ -59,10 +59,10 @@
   /** Metrics reporter that listens on block events and increases metrics counters*/
   private final BlockMetricsReporter mMetricsReporter;
 
-  /** BlockMasterClient, only used to inform the master of a new block in commitBlock */
-  private BlockMasterClient mBlockMasterClient;
-  /** FileSystemMasterClient, only used to inform master of a new file in addCheckpoint */
-  private FileSystemMasterClient mFileSystemMasterClient;
+  /** WorkerBlockMasterClient, only used to inform the master of a new block in commitBlock */
+  private WorkerBlockMasterClient mWorkerBlockMasterClient;
```

Comment:
Could you rename this to `mBlockMasterClient`, worker seems redundant since this is the worker package.